Add cite info (#47)
open-mmlab/mmocr: README.md

This project is released under the [Apache 2.0 license](LICENSE).

## Citation

If you find this project useful in your research, please consider cite:

```bibtex
@misc{mmocr2021,

>     title={MMOCR: OpenMMLab Text Detection, Recognition and Understanding Toolbox and Benchmark},

Reviewer: ```suggestion
    title={MMOCR:  A Comprehensive Toolbox for Text Detection, Recognition and Understanding},
```
We are planning to write a technique report named as above for this repo.
